MAINTAINERS.yml

```diff
@@ -499,6 +499,13 @@ Documentation Infrastructure:
   labels:
     - "area: Documentation Infrastructure"
 
+Release Notes:
+  status: odd fixes
```

Comment:
why not make this maintained and set the release managers for the next release instead?

Reply:
It's certainly an option but I was trying to avoid adding more steps to what the release manager has to do (it's a pretty long list already, changing a couple names is certainly not a big deal but it's one more thing...). Last weeks before release are mostly PR saved searches and triaging so I think the tag should be enough. @laurenmurphyx64 @stephanosio what do you think?

Reply:
no strong opinion on this; though, I agree that one might find `odd fixes` for the "Release Notes" area rather odd.

Reply:
Alright, changed.